trie/trie_reader.go

```diff
@@ -51,6 +51,7 @@ type trieReader struct {
 
 // newTrieReader initializes the trie reader with the given node reader.
 func newTrieReader(stateRoot, owner common.Hash, db NodeReader) (*trieReader, error) {
+	// TODO: explain this logic
```

Comment:
What does this line mean?

Reply:
Seem it's unused comment, I removed and squad to the original commit.